parts/k8s/kubernetescustomscript.sh

```diff
@@ -53,6 +54,10 @@ else
     REBOOTREQUIRED=false
 fi
 
+function testOutboundConnection() {
+    retrycmd_if_failure 20 5 20 nc -v 8.8.8.8 53 || retrycmd_if_failure 20 5 20 nc -v 8.8.4.4 53 || exit $ERR_OUTBOUND_CONN_FAIL
```

Comment:
This looks good! Let's do more aggressive timeout and retry. How does 120 1 5 sound? That should give us between ~2 and ~10 min (depending on the type of error) to achieve a single successful response

Reply:
Sounds good let's try that and get success rate data for a few hours and adjust if needed